src/NoteMouseHandler/PencilMouseHandler.js

```diff
@@ -36,25 +37,24 @@ function actionFactory(e, ctx) {
   return null
 }
 
-function getCursor(e, ctx) {
-  const cpos = getLocal(e)
-  const rect = ctx.getEventsUnderPoint(cpos.x, cpos.y)
-  if (rect) {
-    const pos = pointSub(cpos, rect)
+function getCursor(local, ctx) {
+  const rects = ctx.getEventsUnderPoint(local.x, local.y)
+  if (rects.length > 0) {
+    const rect = rects[0]
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->